src/components/About.tsx

```diff
@@ -0,0 +1,80 @@
+import Image from "./Image";
+import Link from "./Link";
+import NextLink from "next/link";
+
+interface Props extends React.HTMLProps<HTMLDivElement> {
+  wowMadeEasy?: string;
+}
+
+export const About = ({
+  className = "",
+  wowMadeEasy = "https://www.wowmadeeasy.com/",
```

Comment:
considering it's hardcoded anyway I don't see why this should really be a prop

Reply:
Yeah it's a bit moot innit. Trimmed it off.